src/@next/EmptyState/EmptyState.tsx

```diff
@@ -36,20 +37,35 @@ export const EmptyState = ({
   fullWidth = true,
   helpText,
   imageName,
-  customImage,
+  image,
   title,
 }: EmptyStateProps) => {
   const displayButtons = !!primaryButtonAction || !!basicButtonAction;
 
-  const isValidImageName = imageNames.includes(imageName);
-  if (imageName && !isValidImageName) {
-    console.warn(`imageName "${imageName}" is not a valid Image Name.`);
+  const isValidImageName = imageNames.includes(image as ImageName);
+
+  if (typeof image === 'string' && !isValidImageName) {
+    console.warn(`image "${imageName}" is not a valid Image Name.`);
+  }
+
+  if (imageName) {
+    console.warn(
+      'imageName will soon be deprecated. Please use image prop instead'
+    );
   }
 
+  const renderImage = () => {
+    if (typeof image === 'string') {
```

Comment:
this is still not backwards compatible, u need to check if imageName is present and then do the following as well